docs/adr/0003-content-organization-and-accesses.rst

```diff
@@ -0,0 +1,43 @@
+ADR 0003 - Content organization and accesses
+============================================
+
+Status
+------
+
+Proposed
+
+
+Context
+-------
+
+We have `actors <0001-actors>`_, and `videos <0002-videos-languages>`_, that we'll call for now "content". We want this content to be organized for actors to manage/view them.
+
+
+Decision
+--------
+
+Videos are grouped in playlists, which is a Django model named ``Playlist``. A playlist belongs to an organization (``Organization`` model defined in `actors <0001-actors>`_) and is created by someone who can be a manager of this organization, or an author who belongs to this organization. This link is materialized by an ``author`` field on the ``Video`` model, a ``ForeignKey`` to the ``User`` model.
+
+The manager can allow many authors to manage a playlist, so there is a ``ManyToMany`` field on the ``Video`` model, named ``editors``, pointing to the ``User`` model.
```

Comment:
many actors not authors right? There is only one author for a video

Reply:
right.